test/nbrowser/Pages.ts

```diff
@@ -126,6 +144,7 @@ describe('Pages', function() {
 
   it('should not change page when clicking the input while renaming page', async () => {
     // check that initially People is selected
+    await driver.findContent('.test-treeview-itemHeader', 'People').doClick();
```

Comment:
I think this is the problem. I suggest adding a little helper at the end:

```
function clickPage(name: string|RegExp) {
  return driver.findContent('.test-treeview-itemHeader', name).find(".test-docpage-initial").doClick();
}
```

And replacing all lines that want to change the page with a call to that helper.
Tests are sensitive to OS that they are run at. I think on OSX and Windows they click at the top left corner, while on linux they click at the center (or other way around :) ).
That is why you see the difference, as they clicking on the label instead of the initials.